Lib/test/test_memoryview.py

```diff
@@ -679,5 +681,24 @@ def test_picklebuffer_reference_loop(self):
         self.assertIsNone(wr())
 
 
+@threading_helper.requires_working_threading()
+class RacingTest(unittest.TestCase):
+    def test_racing_getbuf_and_releasebuf(self):
+        """Repeatly access the memoryview for racing."""
+        n = 100
+
+        obj = multiprocessing.shared_memory.ShareableList("Uq..SeDAmB+EBrkLl.SG.Z+Z.ZdsV..wT+zLxKwdN\b")
```

Comment:
Can you perhaps add a comment about what this scrambled data represents?

Reply:
It represents nothing, as far as I know. It's just fusil's fuzzer garbage that was in the reproducer.

Reply:
Then maybe add a comment on this. And if you're using local imports, we could save the line length. In addition, I think we should use a SharedMemoryManager to create the shareable list so that we can also change the start method.

Reply:
Hi @picnixz ,

Thanks for your suggestion! 😊

I have updated with `SharedMemoryManager` and use a `range(100)` to replace the scrambled data copied from issue.

Best Regards,
Edward